app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -98,6 +100,17 @@ class ChatActivity : AppCompatActivity(), IChatView {
                 chatPresenter.startComputingThread()
             }
         }
+        
+        val cardViewWithK = findViewById<CardView>(R.id.with_keyboard)
+        val cardViewWithoutK = findViewById<CardView>(R.id.without_keyboard)
+        val keyboard = findViewById<View>(R.id.btn_keyboard)
+        keyboard.setOnClickListener {
+            if(!input_held){
```

Comment:
Spaces around the parentheses missing. Please reformat the code for the changes made.